### PR TITLE
Identity UI updates

### PR DESCRIPTION
**Pull request by eanders-MS**
### Summary
* Update experiment description and add link to feedback.
* Add "experimental feature ahead!" warning to sign in dialog.
* When a user deletes their account, convert cloud projects to local.
* Update delete account to mention the above behavior.
* Add feedback link on My Profile screen.

### Feedback panel
image

### Updated delete account verbiage
image

### Warning on sign in
image

### CODE REVIEW

File: webapp/src/auth.ts [hunk 1]
```diff
@@ -394,13 +395,21 @@ export async function updateUserProfile(opts: {
...
     pxt.storage.removeLocal(CSRF_TOKEN);
 
+    // Convert cloud-saved projects to local projects.
+    await cloud.convertCloudToLocal(userId);
+
     // Update state and UI to reflect logged out state.
     clearState();
+
+    // Reload page
+    window.location.reload();
```
Comment (eanders-MS) [inline webapp/src/auth.ts:412]: This reload can be removed once the project list refresh issue is fixed.

File: theme/common.less [hunk 2]
```diff
@@ -1713,6 +1713,11 @@ div.simframe.ui.embed {
     div.description {
         line-height: 1.1em;
         margin-bottom: 20px;
+
+        .warning {
```
Comment (darzu) [inline theme/common.less:1717]: don't we have css for standard warnings? Is this special somehow?

File: webapp/src/cloud.ts [hunk 3]
```diff
@@ -342,6 +342,23 @@ export function onChangesSynced(changes: Header[]) {
...
+        const localCloudHeaders = workspace.getHeaders(true)
+            .filter(h => h.cloudUserId && h.cloudUserId === userId);
+        const tasks: Promise<void>[] = [];
+        localCloudHeaders.forEach((h) => {
+            // Clear cloud header and re-save the header.
+            delete h.cloudCurrent;
+            delete h.cloudLastSyncTime;
+            delete h.cloudUserId;
+            delete h.cloudVersion;
+            tasks.push(workspace.saveAsync(h));
```
Comment (darzu) [inline webapp/src/cloud.ts:356]: saveAsync should be called like: `saveAsync(h, null, true)` in this case. The last parameter, fromCloudSync, indicates this isn't a user change and we shouldn't touch the timestamps on the header.

File: theme/common.less (hunk 2 above)
Comment (eanders-MS) [reply theme/common.less:1717]: Yes, probably, but this warning message is temporary and will be removed once the feature is out of experiment. If it were going to be long-lived, I would for sure want to use a standard css class.

File: theme/common.less [hunk 4]
```diff
@@ -1805,6 +1815,15 @@ div.simframe.ui.embed {
                 justify-self: start;
                 max-width: 100%;
             }
+
+            div.row2 {
```
Comment (darzu) [inline theme/common.less:1819]: I don't suppose there is a more descriptive name we could use?

**Review by darzu — APPROVED** [review]
LGTM

File: theme/common.less (hunk 4 above)
Comment (eanders-MS) [reply theme/common.less:1819]: That's fair :)

**Pull request merged**